precompiles/parachain-staking/src/lib.rs

```diff
@@ -43,59 +43,59 @@ type BalanceOf<Runtime> = <<Runtime as pallet_parachain_staking::Config>::Curren
 #[derive(Debug, PartialEq)]
 enum Action {
 	// DEPRECATED
-	MinNomination = "min_nomination()",
```

Comment:
@crystalin Can we remove these deprecated functions?

Reply:
If it has been deprecated for 6 month yes

Reply:
It seems to have been deprecated here https://github.com/PureStake/moonbeam/pull/810 8 months ago - I'll remove it from the solidity interface first perhaps (since that is non-binding) and in a follow up remove it from the pallet + precompile.